x/wasm/keeper/test_common.go

```diff
@@ -722,6 +723,7 @@ var keyCounter uint64
 
 // we need to make this deterministic (same every test run), as encoded address size and thus gas cost,
 // depends on the actual bytes (due to ugly CanonicalAddress encoding)
+//
```

Comment:
unnecessary comment fix?

Reply:
it was modified by goimports (golangci-lint)